lib/rake/file_task.rb

```diff
@@ -19,10 +19,10 @@ def needed?
 
     # Time stamp for file task.
     def timestamp
-      if File.exist?(name)
-        File.mtime(name.to_s)
-      else
-        Rake::LATE
+      begin
```

Comment:
We can possibly get rid of the ! File.exist?(name) in the needed? function above too.